src/uct/ugni/udt/ugni_udt_iface.c

```diff
@@ -189,6 +189,8 @@ static ucs_status_t uct_ugni_udt_iface_query(uct_iface_h tl_iface, uct_iface_att
     iface_attr->latency.growth         = 0;
     iface_attr->bandwidth              = pow(1024, 2); /* bytes */
     iface_attr->priority               = 0;
+    iface_attr->max_num_eps            = iface->super.super.config.max_num_eps;
```

Comment:
maybe add function uct_base_iface_query()?

Reply:
yep, this was my idea - but it could do only:
```
memset(iface_attr, 0, sizeof(*iface_attr));
iface_attr->max_num_eps = iface->config.max_eps;
```